examples/multiple_servo_pdo_tuning.py

```diff
@@ -0,0 +1,285 @@
+import time
+from dataclasses import dataclass
+from typing import Union
+
+import matplotlib.pyplot as plt
+from ingenialink.pdo import RPDOMapItem, TPDOMapItem
+from ingeniamotion.enums import FilterType, OperationMode
+from ingeniamotion.motion_controller import MotionController
+
+SRV_1: str = "1"
+SRV_2: str = "2"
+
+PDO_REFRESH_RATE: float = 0.001  # seconds
+CAPTURE_TIME: int = 10  # seconds
+
+SQUARE_WAVE_AMPLITUDE_A: float = 5
+SQUARE_WAVE_AMPLITUDE_B: float = -5
+SQUARE_WAVE_FREQUENCY: float = 0.1
+SQUARE_WAVE_DUTY_CYCLE: float = 0.5
+
+TORQUE_SET_POINT_REGISTER: str = "CL_TOR_SET_POINT_VALUE"
+POSITION_SET_POINT_REGISTER: str = "CL_POS_SET_POINT_VALUE"
+VELOCITY_SET_POINT_REGISTER: str = "CL_VEL_SET_POINT_VALUE"
+
+VELOCITY_ACTUAL_REGISTER = "CL_VEL_FBK_VALUE"
+POSITION_ACTUAL_REGISTER = "CL_POS_FBK_VALUE"
+TORQUE_ACTUAL_REGISTER = "CL_TOR_FBK_VALUE"
+
+
+def configure_servo(operation_mode: OperationMode, mc: MotionController, servo: str) -> None:
+    """Configure the servo with the initial configuration."""
+
+    mc.motion.set_operation_mode(operation_mode, servo)
+    mc.communication.set_register(VELOCITY_SET_POINT_REGISTER, 0, servo)
+    mc.communication.set_register(TORQUE_SET_POINT_REGISTER, 0, servo)
+
+
+class SquareWave:
+    """Class to create a square wave signal."""
+
+    def __init__(
+        self,
+        amplitude_a: float,
+        amplitude_b: float,
+        frequency: float,
+        duty_cycle: float,
+        phy: float = 0,
+    ) -> None:
+        self.amplitude_a: float = amplitude_a
+        self.amplitude_b: float = amplitude_b
+        self.frequency: float = frequency
+        self.time_period: float = 1 / frequency
+        self.current_time: float = 0
+        self.duty_cycle: float = duty_cycle
+        self.phy: float = phy
```

Comment:
delay?
If you mean the greek letter is "phi"

Reply:
Oops, maybe I'll change it to 'phase_shift', I think it's clearer.